.github/workflows/flake8.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        pip install flake8
+        flake8 . --count --show-source --statistics
```

Comment:
Installing flake8 with pip can be an independent job. Doing this enables you to know an error is occurred either on installing flake8 or linting the code.

Reply:
You may refer to this page: https://docs.github.com/ja/actions/language-and-framework-guides/using-python-with-github-actions#starting-with-the-python-workflow-template

Reply:
resolved